src/inc/quic_platform.h

```diff
@@ -66,17 +66,44 @@ typedef struct QUIC_SINGLE_LIST_ENTRY {
 //
 // Different pool tags used for marking allocations.
 //
+#ifdef _WIN32
+
+#define POOL_TAG_QUIC 'CIUQ'
+#define POOL_TAG_QCon 'noCQ'
+#define POOL_TAG_QCTP 'PTCQ'
+#define POOL_TAG_QStm 'mtSQ'
+#define POOL_TAG_QSBf 'fBSQ'
+#define POOL_TAG_QSFM 'MFSQ'
+#define POOL_TAG_QDta 'atDQ'
+#define POOL_TAG_QTst 'tsTQ'
+#define POOL_TAG_QPrf 'frPQ'
+#define POOL_TAG_QTol 'loTQ'
+
+#define QUIC_POOL_TAG(a, b, c, d) POOL_TAG_ ##a ## b ## c ## d
```

Comment:
Not  a super fan of this. Is there no way to concatenate single quoted characters?

Reply:
Nope. See https://gcc.gnu.org/onlinedocs/cpp/Stringizing.html, which says:
> There is no way to convert a macro argument into a character constant.

Reply:
We could also decide to just leave the `-Wno-multichar` in place...

Reply:
Ok.